test/MyNypl.test.js

```diff
@@ -147,15 +147,15 @@ describe('MyNypl', () => {
     });
 
     it('should have the <a> with class name "MyNypl-Catalog-Btn". Its text equals ' +
-      '"GO TO THE CATALOG"', () => {
+      '"Log in to your accountGO TO THE CATALOG"', () => {
       expect(component.find('.MyNypl-Catalog-Btn').text())
-        .to.equal('GO TO THE CATALOG');
+        .to.equal('Log in to your accountGO TO THE CATALOG');
```

Comment:
probably easier to update unit tests if we use `.to.contain(stringMatch)` -- I see why it fails with a strict equal check but what is really being tested is not the SVG title/property but the text of the button.

See: https://github.com/NYPL/staff-picks/blob/master/test/unit/Book.test.js#L98

Reply:
Yeah... I kinda prefer strict equality checks.